app/views/fragments/promotion/guardianWeeklyLegalTerms.scala.html

```diff
@@ -3,39 +3,94 @@
 @import com.gu.memsub.promo.Promotion.AnyPromotion
 @import com.gu.memsub.subsv2.Catalog
 @import views.support.MarkdownRenderer
+@import com.gu.i18n.{Country, CountryGroup}
 
-@(promoCode: PromoCode, catalog: Catalog, promotion: AnyPromotion, md: MarkdownRenderer)
+@(promoCode: PromoCode, catalog: Catalog, promotion: AnyPromotion, md: MarkdownRenderer, maybeCountry: Option[Country])
 @isSixForSix = @{(catalog.weekly.plans.flatten.filter(_.charges.billingPeriod == SixWeeks).map(_.id).toSet intersect promotion.appliesTo.productRatePlanIds).nonEmpty}
-@Html(md.render(
-    (if (isSixForSix) {
-s"""
-Offer not available to current subscribers of Guardian Weekly. You must be 18+ to be eligible for this offer. Guardian Weekly reserve the right to end this offer at any time.
 
-**United Kingdom:** Offer is £6 for the first 6 issues followed by quarterly (13 weeks) subscription payments of £30 thereafter, saving 20% off the cover price.
+@uk = @{"**United Kingdom:**"}
+@ukQuarterlyRate = @{"£37.50"}
```

Comment:
What is the logic behind splitting out all these variables rather than just inlining them in the terms below? It seems to add quite a lot of noise

Reply:
So that when they're changed they're changed in one place and reduces the chances of typos when editing.